node-server/src/database/dao.js

```diff
@@ -168,13 +188,14 @@ class Dao {
    * @returns {Promise}
    */
   getLastMessageId(nodeId) {
-    return this.db.executeQuery('all', `SELECT MAX(id)
+    return this.db.executeQuery('get', `SELECT MAX(id) AS 'maxId'
       FROM messages WHERE node_id = :nodeId`, [nodeId])
   }
 
   getLastMessageIds() {
-    return this.db.executeQuery('all', `SELECT node_id, MAX(id) 
-      FROM messages 
+    return this.db.executeQuery('all', `SELECT node_id AS 'nodeId',
+      MAX(id) AS 'id'
```

Comment:
This requires adjusting the synchronizer. Currently, the requests are empty because the keys have changed